geode-core/src/main/java/org/apache/geode/internal/cache/wan/GatewaySenderEventImpl.java

```diff
@@ -774,6 +776,15 @@ public String toString() {
     return buffer.toString();
   }
 
+  public String toSmallString() {
+    StringBuffer buffer = new StringBuffer();
```

Comment:
I think you want to use 'StringBuilder' here to avoid synchronization overhead from 'StringBuffer'.

Reply:
Thanks for the review. I meant to use StringBuilder. I guess I picked the wrong auto-completion.

Reply:
Oh, I see what I did here. I just copied the existing toString method and modified it. I'll fix that method too.